packages/orchestration/src/exos/chain-hub.js

```diff
@@ -463,6 +511,40 @@ export const makeChainHub = (zone, agoricNames, vowTools) => {
         brandDenoms.init(detail.brand, denom);
       }
     },
+    /**
+     * Update asset info by completely replacing existing entry
+     *
+     * @param {Denom} denom - Denom on the holding chain
+     * @param {DenomDetail} detail - New asset details
+     * @throws {Error} If asset not registered or referenced chains not
+     *   registered
+     */
+    updateAsset(denom, detail) {
```

Comment:
since it's replacing the entry `replaceAsset` ?

Reply:
if we want `replace*`, we should make the same change on the `*connection`, `*chain` methods

Reply:
`updateX` is consistent with `nameAdmin.update(k, v)`. seems ok to me